pydantic/fields.py

```diff
@@ -962,6 +962,8 @@ class ComputedFieldInfo:
         alias_priority: priority of the alias. This affects whether an alias generator is used
         title: Title of the computed field as in OpenAPI document, should be a short summary.
         description: Description of the computed field as in OpenAPI document.
+        examples: Example values of the computed field as in OpenAPI document.
+        json_schema_extra: Dictionary of extra JSON schema properties.
```

Comment:
Throughout this file, I think we should update the description associated with `json_schema_extra` to reflect the fact that it can be a dict or `Callable`

Reply:
The documentation in general barely mentions the callable part and doesn't explain it, how about a separate issue for that? The scope of this PR has already crept quite a bit.

Reply:
Yeah, that's fine.